salt/metalk8s/repo/installed.sls

```diff
@@ -61,12 +66,13 @@ Install package repositories manifest:
       - {{ nginx_registry_config_path }}
     - context:
         container_port: {{ repo.port }}
-        image: {{ package_repositories_image }}
+        image: docker.io/library/nginx:1.15.8
```

Comment:
Could take the `alpine` image for this, since it's uncompressed - just append `-alpine`. Isofunctional as far as a `doit vagrant_up` is concerned.

Reply:
As discussed on the channel, we should try to be compliant and use CentOS, so no Alpine.